src/main/java/com/coveros/selenified/utilities/TestCase.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright 2018 Coveros, Inc.
+ *
+ * This file is part of Selenified.
+ *
+ * Selenified is licensed under the Apache License, Version
+ * 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.coveros.selenified.utilities;
+
+import java.lang.reflect.Method;
+import java.util.Random;
+
+/**
+ * Manages the test cases, including naming conventions and formatting
+ *
+ * @author Max Saperstone
+ * @version 3.0.4
+ * @lastupdate 1/12/2019
+ */
+public class TestCase {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1118)